src/components/notes/offline-indicator.styl

```diff
@@ -0,0 +1,26 @@
+.grid
+    padding 1rem
+    font-size 1rem
+    .banner &
+        font-size 0.875rem
+        line-height 1.3
+        padding 0.25rem 1rem 0.25rem 1rem
+        margin 0
+        width 100%
+
+.chip
+    background-color var(--silver)
+    opacity 60%
+
+.tooltip
+    background-color var(--paleGrey) !important
+    border-radius 8px !important
+    box-shadow 0 1px 3px 0 rgba(50, 54, 63, 0.19), 0 6px 18px 0 rgba(50, 54, 63, 0.19)
+    color var(--black) !important
+    line-height 1.3 !important
+    position relative
+    left -1rem
```

Comment:
Generally I don't like to have a negative value. Specially when using a tooltip and a popover component. Why do you need it?

Reply:
Je suis passé autrement (vu que j'ai utilisé le CSS in JS plus haut, j'ai pu changer le positionnement à la source)